tests/specs/blog_v2/migrations/m200000_000004_create_table_posts2tags.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * Table for Posts2Tags
+ */
+class m200000_000004_create_table_posts2tags extends \yii\db\Migration
+{
+    public function up()
+    {
+        $this->createTable('{{%posts2tags}}', [
+            'id' => $this->bigPrimaryKey(),
+            'post_id' => $this->bigInteger()->notNull(),
+            'tag_id' => $this->bigInteger()->notNull(),
+        ]);
```

Comment:
junction table primary key should be composite PK: `PRIMARY KEY(post_id, tag_id)` `id` does not make sense here.

Reply:
done